package.json

```diff
@@ -75,6 +75,11 @@
                 "category": "ev3dev"
             },
             {
+                "command": "ev3devBrowser.captureScreenshot",
+                "title": "Take Screenshot",
+                "category": "ev3dev"
```

Comment:
OK, it looks like assigning `"category"` here causes this command to show up in the command palette. Let's omit this line for now.